docs/api.md

```diff
@@ -891,6 +892,18 @@ stream:                 -2-4-5-6-8->
 stream.skipWhile(even): -----5-6-8->
 ```
 
+### skipAfter
+
+#### `stream.skipAfter(predicate) -> Stream`
+#### `most.skipAfter(predicate, stream) -> Stream`
+
+Create a new stream containing all events before and including when the `predicate` returns true.
+
+```
+stream:                 -1-2-4-5-6-8->
+stream.skipAfter(even): ---1-2|
```

Comment:
Couple quick suggestions.  The docs here are showing a delay:

```
stream:                 -1-2-4-5-6-8->
stream.skipAfter(even): ---1-2|
```

Let's align the values to show that there's no delay:

```
stream:                 -1-2-4-5-6-8->
stream.skipAfter(even): -1-2|
```

Someone might wonder why 3 is missing from the example.  The result is the same if we include 3, so might be worth it just to remove a possible wtf (probably rare) for readers.

In which case, we'd end up with:

```
stream:                 -1-2-3-4-5-6-8->
stream.skipAfter(even): -1-2|
```